include/c3d/renderqueue.h

```diff
@@ -59,21 +118,95 @@ union C3D_DEPTHTYPE
+ */
 C3D_RenderTarget* C3D_RenderTargetCreateFromTex(C3D_Tex* tex, GPU_TEXFACE face, int level, C3D_DEPTHTYPE depthFmt);
+
+/**
+ * @brief Deletes render target.
+ * @param[in] target Pointer to render target.
+ */
 void C3D_RenderTargetDelete(C3D_RenderTarget* target);
+
+/**
+ * @brief Sets render target output to screen.
+ * @param[in] target        Pointer to render target.
+ * @param[in] screen        Screen to transfer the framebuffer to.
+ * @param[in] side          Side of the screen to transfer the framebuffer to. (unused for the bottom screen)
+ * @param[in] transferFlags Specifies GX_TRANSFER bitflags.
+ */
 void C3D_RenderTargetSetOutput(C3D_RenderTarget* target, gfxScreen_t screen, gfx3dSide_t side, u32 transferFlags);
 
+/**
+ * @brief Detaches render target from screen output.
+ * @param[in] target Pointer to render target.
+ */
 static inline void C3D_RenderTargetDetachOutput(C3D_RenderTarget* target)
 {
 	C3D_RenderTargetSetOutput(NULL, target->screen, target->side, 0);
 }
 
+/**
+ * @brief Clears framebuffer of target.
+ * @param[in] target     Pointer to render target.
+ * @param[in] clearBits  Specifies which buffers to clear. (see \ref C3D_ClearBits)
+ * @param[in] clearColor 32 bit RGBA value to clear color buffer with.
+ * @param[in] clearDepth Value to clear depth buffer with.
+ */
 static inline void C3D_RenderTargetClear(C3D_RenderTarget* target, C3D_ClearBits clearBits, u32 clearColor, u32 clearDepth)
 {
 	C3D_FrameBufClear(&target->frameBuf, clearBits, clearColor, clearDepth);
 }
 
+/**
+ * @brief Synchronizes and initiates a display transfer.
+ * Synchronizes and initiates a \ref GX_DisplayTransfer().
+ * @param[in]  inadr  Address of the input.
+ * @param[in]  indim  Dimensions of the input.
+ * @param[out] outadr Address of the output.
+ * @param[in]  outdim Dimensions of the output.
+ * @param[in]  flags  Flags to transfer with.
+ */
 void C3D_SyncDisplayTransfer(u32* inadr, u32 indim, u32* outadr, u32 outdim, u32 flags);
+
+/**
+ * @brief Synchronizes and initiates a texture copy.
+ * Synchronizes and initiates a \ref GX_TextureCopy().
+ * @param[in]  inadr  Address of the input.
+ * @param[in]  indim  Dimensions of the input.
+ * @param[out] outadr Address of the output.
+ * @param[in]  outdim Dimensions of the output.
+ * @param[in]  size   Size of the data to transfer.
+ * @param[in]  flags  Flags to transfer with.
+ */
 void C3D_SyncTextureCopy(u32* inadr, u32 indim, u32* outadr, u32 outdim, u32 size, u32 flags);
+
+/**
+ * @brief Synchronizes and fills the memory of two buffers with the given values.
+ * Synchronizes and initiates a \ref GX_MemoryFill().
+ * @param[in] buf0a    Start address of the first buffer.
+ * @param[in] buf0v    Dimensions of the first buffer.
+ * @param[in] buf0e    End address of the first buffer.
+ * @param[in] control0 Value to fill the first buffer with.
+ * @param[in] buf1a    Start address of the second buffer.
+ * @param[in] buf1v    Dimensions of the second buffer.
```

Comment:
are these (and other `u32` "dimension" parameters) in the `C3D_IVec` format shown in `types.h`? It might be useful to note that (or simply change the type to reflect that more clearly, since it looks like it's just a typedef of `u32`).

Reply:
No.
https://libctru.devkitpro.org/gx_8h.html#ad34237bd30a2a96a6074e8967840d4fc
https://github.com/Swiftloke/ModMoon/blob/bc32168ac060cca96ca45f075becce5118d1ba5f/source/sdraw.cpp#L131

Reply:
Err, sorry. In this context it is the fill value for the buffer.